src/GregClient/Responses/Responses.cs

```diff
@@ -125,7 +125,7 @@ public class PackageVersion
 
         public string name { get; set; }
 
-        public string id { get; set; }
+        public string _id { get; set; }
```

Comment:
`name` and `id` have not yet been consumed by Dynamo...so it should safe to change